### DISCUSSION TITLE
Definir el análisis más específico

### DISCUSSION THREAD

**Discussion by solmos** [PaulaMorBla14/TFG-Baloncesto#8] [Ideas]
Veo que mencionas aquí el índice Gini. Si vas a hacer el análisis del más-menos ajustado con ridge regression creo que es más que suficiente ya que creo que te va a llevar más tiempo del que crees.

https://github.com/PaulaMorBla14/TFG-Baloncesto/blob/e706cf14d27adceebf21ec3934363fb9cce801ed/Project_v1.Rmd#L33

Piensa bien en qué análisis quieres realizar y centra el trabajo en torno a él. Si necesitas que te ayudemos a definir el análisis o a aclarar algunas partes de este háznoslo saber.

**Comment by PaulaMorBla14** [comment]
Bueno, esta parte que escribí de `#Conceptos` no es parte del trabajo como tal. Quiero decir, apunté todo lo que comentamos en la anterior reunión para hacerme una idea de proyecto pero es un borrador, luego lo acabaré de definir una vez tenga el trabajo acabado.
